manifest.yml

```diff
@@ -1,6 +1,6 @@
 ---
 declared-services:
-  my-conversation-service:
```

Comment:
if you use `watson-assistant-service` above then you need to update this to be the same

Reply:
yes had fixed this